code/modules/mob/living/simple_animal/hostile/megafauna/demonic_frost_miner.dm

```diff
@@ -265,6 +272,8 @@ Difficulty: Extremely Hard
 		return
 	var/turf/T = get_turf(src)
 	var/loot = rand(1, 3)
+	for(var/obj/structure/frost_miner_prism/P in GLOB.frost_miner_prisms)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names